src/clojure_tictactoe/cli/output/board_printer.clj

```diff
@@ -0,0 +1,20 @@
+(ns clojure-tictactoe.cli.output.board-printer)
+
+(defn render-board [spaces]
+ (str  " " (spaces 0) " | " 
+      (spaces 1) " | " 
+      (spaces 2) 
+       " \n===+===+===\n " 
+      (spaces 3) " | " 
+      (spaces 4) " | " 
+      (spaces 5) 
+      " \n===+===+===\n " 
+      (spaces 6) " | " 
+      (spaces 7) " | " 
+      (spaces 8) " \n"))
+
+(defn render-board-spaces
```

Comment:
There's a bit of responsibility clash here. `render-board-spaces` is responsible for renting a board and taking in a newly updated board and rendering that. This responsibility could be seperated out into two different functions, I think. 

For example `fill-space [board]` whereas `fill-space` takes a board, updates it, then returns it.

And you'd have your `defn construct-board` or something similar, that would simply have the responsibility of building the board.